### ISSUE TITLE
wLED backup issues with new wLED version.

### ISSUE THREAD

**Issue by SirGoodenough** [danmed/TasmoBackupV1#73]
I know you have MANY limitations dealing with wLED.  Not here to ask for more function.
What I noticed today is I did a backup of a device with the latest release on it was an issue with the existing code.
This is the device that I backed up.

![Screenshot_20220922_185313](https://user-images.githubusercontent.com/47349533/191870598-33cbf46f-eda7-4770-b2cc-2e6688345246.png)

The back-up appears to be a success as reported by Tasmobackup.

After it does so, however the reported version has not changed...

![Screenshot_20220922_185541](https://user-images.githubusercontent.com/47349533/191870763-e27fa69f-80b1-4db9-9562-6c388b1a4fe9.png)

As long as there is a backup there, it's all good.  Just point it out as they probably changed some variable on you again.

**Comment by patrickdk77** [comment]
Ya, backups are failing. I did test this against 0.13.0-bx and 0.11 and I think 0.12
But testing it again now, using 0.13.1 and 0.13.3, all backups in wled seem to be broken.

If I click backup config or backup presets from the wled settings page, they also fail for me, so it doesn't look like I can fix this.

**Comment by devzwf** [comment]
i think that fixed 
i can manually backup config and presets with no issue for the WebUi of wled

**Comment by lensherm** [comment]
Confirming that I can successfully backup config and presets from the WLED GUI on the following versions:
0.13.3
0.13.1 SR fork

Clicking the  Backup button on individual WLED devices in TasmoBackup 1.05.00 results in a "Backup completed successfully!" message, yet after this, no backup files are listed for these devices.

**Comment by patrick-blom** [comment]
Can confirm the same behavior @lensherm described.

**Comment by patrickdk77** [comment]
Yes, I need to find time to retest this, for me the wled gui wasn't giving me backups, could have been anything that caused that issue, but it was stopping me from being able to reproduce this correctly, to find the real issue. I won't have time to look into this though till probably thanksgiving. Hope to put out a nice fresh release then fixing a few things.

**Comment by patrick-blom** [comment]
I found the issue and created a quick fix.

Plz have a look at my PR: https://github.com/danmed/TasmoBackupV1/pull/82

Cheers Patrick
